account_invoice_section_sale_order/tests/common.py

```diff
@@ -1,13 +1,12 @@
 from odoo.tests import tagged
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
 @tagged("-at_install", "post_install")
-class Common(SavepointCase):
+class Common(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
this should be removed only if `BaseCommon` class is used instead of `TransactionCase`